Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/dialogController/ImportController.java

```diff
@@ -308,7 +316,7 @@ private void uploadFileFromURL(String fileUrl) throws ExceptionImport, Exception
             }
             okButton_URL.setDisabled(false);
 
-        } catch (MalformedURLException e) {
+        } catch (URISyntaxException | MalformedURLException e) {
```

Comment:
catch this inside isValidCloudStorageURL log it and return false and through the